pkg/kubelet/network/plugins.go

```diff
@@ -38,6 +38,13 @@ type NetworkPlugin interface {
 	// before any other methods are called.
 	Init(host Host) error
 
+	// Tells the plugin to start any long-running operations; the plugin
+	// should not block this function
+	Run() error
+
+	// Indicates to the plugin that the pod CIDR changed, if controlled by kubelet
+	PodCIDRChanged(cidr string)
```

Comment:
@thockin that's a great question; I was copying the existing cbr0 and shaping code here, which does apparently allow PodCIDR changes.  Unless somebody makes a call that we actually don't care about those changes, removing this from the plugin API would be a regression in kubenet versus integrated cbr0 code.

Who can make that call?  I'm always happy to rip code out :)